include/rangers-api/Application/Save/SaveDataAccessor.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+namespace app::save {
+    template<typename Data>
+    class SaveDataAccessor {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.